SwiftTweaks/Tweak.swift

```diff
@@ -76,6 +78,24 @@ extension Tweak where T: SignedNumber {
 	}
 }
 
+extension Tweak {
+    
+    public static func optionsList(_ collectionName: String, _ groupName: String, _ tweakName: String, options: [String], defaultValue: String? = nil) -> Tweak<StringOption> {
+        precondition(!options.isEmpty, "Options list cannot be empty (optionsList tweak \(tweakName)")
+        precondition(
+            defaultValue == nil || (defaultValue != nil && options.index(of: defaultValue!) != nil), "The default value \(defaultValue) of the stringList tweak \(tweakName) must be in the list of options \(options) "
+        )
+        
+        return Tweak<StringOption>(
+            collectionName: collectionName,
+            groupName: groupName,
+            tweakName: tweakName,
+            defaultValue: StringOption(value: defaultValue ?? options.first!),
```

Comment:
It can/should never be nil if `optionsList` type is used. We have  precondition during initialization.